main.go

```diff
@@ -32,6 +32,7 @@ func main() {
 	e := cache.NewEnsureSpacer(0.95, 0.5)
 	h := cache.NewHTTPCache(*dir, *maxSize*1024*1024*1024, e)
 
+	http.HandleFunc("/status", maybeAuth(h.StatusPageHandler, *htpasswd_file, *host))
```

Comment:
I wonder whether the status page should be password protected? What's a good argument for it?

Reply:
I thought it would be less surprising to users / secure by default. Will remove password protection.